bin/localeapp

```diff
@@ -128,9 +128,15 @@ module LocaleappGLIWrapper
   end
 
   desc "Pulls all translations from localeapp.com"
+  arg_name "[target_dir]"
```

Comment:
I was suggesting `(<>)`, not `<>`:
``` ruby
desc "Sends the key and content to localeapp.com"
arg_name "<key> <locale:content> (<locale:content> ...)"
```
but maybe it's a special case, let's forget it.

`[]` is OK as it looks like this is what GLI uses when generating help:
```
SYNOPSIS
    localeapp [global options] command [command options] [arguments...]
```